### PR TITLE
Add new documentation for upcoming release

### PR DESCRIPTION
**Pull request by qpetraroia**
Documentation update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ To properly use Draft to containerize your application, create manifest files, a
 
 -  Supported deployment types: Helm, Kustomize, Kubernetes manifest.
 
+**Build an image on Azure Container Registry** easily build an image on an ACR through the command palette to use for your deployment files.
```
Comment (OliverMKing) [inline README.md:42]: maybe reword to 

"easily build an image on an ACR to later use for your deployment files."

Not sure it's relevant to specify it's through the command palette since they all are.

**Review by OliverMKing — APPROVED** [review]

**Review by OliverMKing — APPROVED** [review]

**Review by davidgamero — APPROVED** [review]
lgtm

**Comment by sabbour** [comment]
Made a small change to remove "easily" (https://justsimply.dev/).
